Arcane Engine Core/src/input/InputManager.h

```diff
@@ -17,6 +17,8 @@ namespace arcane {
 		static bool isKeyPressed(unsigned int keycode);
 		static float getKeyPressure(unsigned int keycode);
 		static bool isMouseButtonPressed(unsigned int code);
+		static bool GetButton(int keyCode);
```

Comment:
Functions should be lower camel cased. I sometimes capitalize functions too so I need to cleanup my code ;)
The same goes for all of the new functions you defined